Userland/Services/LaunchServer/Launcher.cpp

```diff
@@ -156,6 +169,20 @@ Vector<String> Launcher::handlers_with_details_for_url(const URL& url)
     return handlers;
 }
 
+Optional<String> Launcher::mime_type_for_file(String path)
+{
+    auto file_or_error = Core::File::open(path, Core::OpenMode::ReadOnly);
+    if (file_or_error.is_error()) {
+        return {};
+    } else {
+        auto file = file_or_error.release_value();
+        // Read accounts for longest possible offset + signature we currently match against.
+        auto bytes = file->read(0x9006);
```

Comment:
No need to fix this, but we should probably compute that sniff size automatically... I think I just yakbait-ed myself.